llvm/lib/Support/RISCVISAInfo.cpp

```diff
@@ -839,6 +860,33 @@ RISCVISAInfo::parseArchString(StringRef Arch, bool EnableExperimentalExtension,
                              "string must be lowercase");
   }
 
+  bool IsProfile = Arch.starts_with("rvi") || Arch.starts_with("rva") ||
+                   Arch.starts_with("rvb") || Arch.starts_with("rvm");
```

Comment:
> profile-name ::= "RV"`<profile-family-name><profile-ratified-year><privilege-mode><ISA-XLEN>`
> profile-family-name ::= "I" | "M" | "A"

Missing tests for `rvm`.
Do you know what "rvb" stands for?

Reply:
RV[A|B|M]23 haven't been ratified, but I can add them as well.
`RVB` means some low-end APs I think?